ot/optim.py

```diff
@@ -150,7 +170,10 @@ def generic_conditional_gradient(a, b, M, f, df, reg1, reg2, lp_solver, line_sea
     b : array-like, shape (nt,)
         samples weights in the target domain
     M : array-like, shape (ns, nt)
-        loss matrix
+        loss matrixif isinstance(gfk, int) or isinstance(gfk, float):
```

Comment:
what is this doing in the docstring?

Reply:
wth... Thanks Rémi for the review ! Should be ready for merge now :)